bolt/test/X86/cfg_build_hlt.test

```diff
@@ -0,0 +1,13 @@
+## Check CFG for halt instruction
```

Comment:
It's more convenient to combine `*.s` sources with the test in the same file. Check `double-rel.s` for example.